web/src/main/webapp/WEB-INF/jsp/include/showTable.jsp

```diff
@@ -213,13 +213,13 @@
             <%-- Alter header format to reduce table space--%>
             <c:choose>
             	<c:when test="${column.name eq 'Date Updated'}">
-            		<c:out value="Date<br />Updated" escapeXml="false" />
+					<fmt:message key="date_br_updated" bundle="${resword}"/>
             	</c:when>
             	<c:when test="${column.name eq 'Last Updated by'}">
-            		<c:out value="Last<br /> Updated by" escapeXml="false" />
+					<fmt:message key="last_br_updated_by" bundle="${resword}"/>
             	</c:when>
             	<c:when test="${column.name eq 'Date Created'}">
-            		<c:out value="Date<br />Created" escapeXml="false" />
+					<fmt:message key="date_br_created" bundle="${resword}"/>
             	</c:when>
```

Comment:
I got it. I think the one who wrote this just want to make this test for English words and he not sure that other language need the <br/> in the middle. People want the <br/> they will do it themself when they translate the words. I'll revert this change to the original.

Thank you for reviewing!